Better UI Changes

creativeblaq opened the pull request:
Made some changes to the User interface both on android and IOS

jhomlala commented:
Could you explain why did you these changes and what's the purpose of these fixes? Unfortunately, "Better UI" is not enough for me.

creativeblaq commented:
> Could you explain why did you these changes and what's the purpose of these fixes? Unfortunately, "Better UI" is not enough for me.

Simply put no fundamental changes where made to the overall core functionality, only user interface and user experience changes. These are just minor changes to the user controls and how the user interacts with the player.

**Changes made**

![WhatsApp Image 2021-07-08 at 16 43 21 (1)](https://user-images.githubusercontent.com/76699978/125062583-d789dd80-e0ae-11eb-9844-8810dc761071.jpeg)

1. Default player Icons changed
2. Moved progress bar to the bottom to have more touch space for longer videos 
3. This leaves the ui having a more stacked view and breathability  

![WhatsApp Image 2021-07-08 at 16 43 21 (3)](https://user-images.githubusercontent.com/76699978/125062625-e6709000-e0ae-11eb-98ff-4c7f111ea14c.jpeg)

4. Modal dialog switches from the default material modal on android and cupertino popup modal on ios
5. Added some rounded edges for taste
6. Selected modal item has a check icon to clearly identify the user choice

Planning on adding more interface features like double tap to skip and allowing the user to choose whether they want a stacked ui or a more linear.

sha3rawi33 commented:
@creativeblaq That's really nice work. when can you implement the double ab seek? I'm waiting for it 💯 
good luck

kabirnayeem99 commented:
Wow. This looks sick. Loving this so far. Client asked for exactly the same changes a few months ago. And the current controls, honestly, looks kind of ugly. Waiting to see these changes merged into the main branch.

kabirnayeem99 commented on example/lib/pages/playlist_page.dart:
```diff
@@ -30,7 +30,7 @@ class _PlaylistPageState extends State<PlaylistPage> {
     _betterPlayerPlaylistConfiguration = BetterPlayerPlaylistConfiguration(
       loopVideos: true,
-      nextVideoDelay: Duration(seconds: 1),
+      nextVideoDelay: Duration(seconds: 3),
```
I thought the change were only related to the UI.

creativeblaq replied:
To me this did seem like a ui component since there is a display component that tells the user how long till the next video.
If 3 seconds is a lot this is honestly one line of code that can easily be modified.

kabirnayeem99 commented on lib/src/configuration/better_player_controls_configuration.dart:
```diff
@@ -206,15 +207,15 @@ class BetterPlayerControlsConfiguration {
-      this.forwardSkipTimeInMilliseconds = 15000,
-      this.backwardSkipTimeInMilliseconds = 15000,
+      this.forwardSkipTimeInMilliseconds = 10000,
+      this.backwardSkipTimeInMilliseconds = 10000,
```
This is also changes unrelated to UI.

creativeblaq replied:
This has actually changed based on the ui change. The default icon for skipping forward or back says your skipping at a rate of 10 seconds, this however is still modifiable when constructing the player controls configurations instance.

kabirnayeem99 reviewed (commented):
There are some changes that are unrelated to the UK, such as, 
* Change in the delay time in 'nextVideoPlay' have been changed from 1 to 3 seconds. 
* forwardSkipTimeInMilliseconds has also been changed from 15000 to 10000.

creativeblaq commented:
> There are some changes that are unrelated to the UK, such as, 
> * Change in the delay time in 'nextVideoPlay' have been changed from 1 to 3 seconds. 
> * forwardSkipTimeInMilliseconds has also been changed from 15000 to 10000.

I am assuming you meant the UI, but I did mention that the changes I made focused both on UI and UX (User interface and **User experience**), I'm not sure you are aware that these times can be changed when creating the player controls configuration instance in the code and can be passed through as parameters there to tailor them to your needs.

creativeblaq commented:
> @creativeblaq That's really nice work. when can you implement the double tab seek? I'm waiting for it 💯 
> good luck 

I will get to it as soon as possible. I am glad you approve of the changes.

kabirnayeem99 commented:
I am aware of this, sir. Not to mention that, I love this new design. I have just mentioned those changes to clarify. Thank your for your kind reply.

jhomlala commented:
Awesome work. LGTM!

Supriyanto6543 commented:
When you add new features about next and previous video sir? I am waiting it. 
@jhomlala
